portal/server/validation/validate-ids.js

```diff
@@ -74,6 +74,14 @@ const isValidFileName = (value) => FILE_NAME_REGEX.test(value);
  */
 const isValidBankId = (bankId) => isValidRegex(/^\d+$/, bankId);
 
+/**
+ * isValidSqlId
+ * checks if id conforms to regex
+ * @param {String} id
+ * @returns {Boolean} asserts if regex is matched
+ */
+const isValidSqlId = (id) => isValidRegex(/^\d+$/, id);
```

Comment:
Not for this PR (and not linked to any work you've done!), but `isValidRegex` is not a very useful helper, isn't typed correctly and has a misleading name - another piece of tech debt rears its head!